packages/storybook/stories/index.js

```diff
@@ -14,44 +14,49 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 */
+
 import React from 'react';
 import { storiesOf, action } from '@kadira/storybook';
 import { withKnobs, text, select } from '@kadira/storybook-addon-knobs';
-import Button from '../components/button';
+import { Button } from '../../react/lib/2016-12-01';
 import SourceViewer from '../components/source_viewer';
 
+const Example = require('../components/example');
+
 const colorOptions = {
-  '#000000': 'Black',
-  '#ffffff': 'White',
+  black: 'Black',
+  white: 'White',
+  blue: 'Blue',
+
 };
 
 storiesOf('Button', module)
   .addDecorator(withKnobs)
   .add('with text', () => {
     const buttonText = text('Text', 'Hello, button!');
     const props = {
-      backgroundColor: select('Background Color', colorOptions, '#ffffff'),
-      color: select('Color', colorOptions, '#000000'),
+      background: select('Background Color', colorOptions, 'black'),
+      color: select('Color', colorOptions, 'white'),
       onClick: action('clicked'),
     };
 
     const sources = [
       {
         label: 'React',
         source: `
-          <OrionButton
-            backgroundColor={"${props.backgroundColor}"}
+          <Button
+            background={"${props.background}"}
             color={"${props.color}"}
             onClick={action('clicked')}
           >
             {"${buttonText}"}
-          </OrionButton>
+          </Button>
         `,
       }, {
         label: 'Web components',
```

Comment:
I don't think we should document the Web Component variant since it doesn't actually work yet.